editor/src/components/inspector/inspector-strategies/fixed-fill-hug.spec.browser2.tsx

```diff
@@ -131,6 +131,153 @@ describe('Fixed / Fill / Hug control', () => {
+      it('child is set to fill container on the vertical axis', async () => {
+        const editor = await renderTestEditorWithCode(
+          projectWithChildSetToVerticalFill,
+          'await-first-dom-report',
+        )
+        const parent = await select(editor, 'parent')
+        const child = editor.renderedDOM.getByTestId('child')
+
+        expect(child.style.width).toEqual('149px')
+        expect(child.style.height).toEqual('')
+        expect(child.style.flexGrow).toEqual('1')
+
+        const control = (await editor.renderedDOM.findAllByText(FixedLabel))[1]
+
+        mouseClickAtPoint(control, { x: 5, y: 5 })
+
+        const button = (await editor.renderedDOM.findAllByText(HugContentsLabel))[0]
+        mouseClickAtPoint(button, { x: 5, y: 5 })
+
+        expect(parent.style.width).toEqual('700px')
+        expect(child.style.width).toEqual('149px')
+        expect(parent.style.height).toEqual(MaxContent)
+        expect(child.style.height).toEqual('759px')
+      })
+
+      it('child is set to fixed size on the horizontal axis', async () => {
+        const editor = await renderTestEditorWithCode(
+          projectWithChildSetToFixed('row'),
+          'await-first-dom-report',
+        )
+        const parent = await select(editor, 'parent')
+        const child = editor.renderedDOM.getByTestId('child')
+
+        expect(child.style.width).toEqual('302px')
+        expect(child.style.height).toEqual('141px')
+
+        const control = (await editor.renderedDOM.findAllByText(FixedLabel))[0]
+
+        mouseClickAtPoint(control, { x: 5, y: 5 })
+
+        const button = (await editor.renderedDOM.findAllByText(HugContentsLabel))[0]
+        mouseClickAtPoint(button, { x: 5, y: 5 })
+
+        expect(parent.style.width).toEqual(MaxContent)
+        expect(child.style.width).toEqual('302px')
+        expect(parent.style.height).toEqual('759px')
+        expect(child.style.height).toEqual('141px')
+      })
+
+      it('child is set to fixed size on the vertical axis', async () => {
+        const editor = await renderTestEditorWithCode(
+          projectWithChildSetToFixed('column'),
+          'await-first-dom-report',
+        )
+        const parent = await select(editor, 'parent')
+        const child = editor.renderedDOM.getByTestId('child')
+
+        expect(child.style.width).toEqual('302px')
+        expect(child.style.height).toEqual('141px')
+
+        const control = (await editor.renderedDOM.findAllByText(FixedLabel))[1]
+
+        mouseClickAtPoint(control, { x: 5, y: 5 })
+
+        const button = (await editor.renderedDOM.findAllByText(HugContentsLabel))[0]
+        mouseClickAtPoint(button, { x: 5, y: 5 })
+
+        expect(parent.style.width).toEqual('700px')
+        expect(child.style.width).toEqual('302px')
+        expect(parent.style.height).toEqual(MaxContent)
+        expect(child.style.height).toEqual('141px')
+      })
+
+      it('child is set to hug contents on the horizontal axis', async () => {
```

Comment:
'child is set to hug contents on the horizontal axis, no conversion happens'

Reply:
added in https://github.com/concrete-utopia/utopia/pull/3160/commits/fb5b8bdea4b1b29ffceb7a288182d1d697824255